asr-python-advanced-finetune-am-conformer-ctc-tao-finetuning-librispeech-nigerian.ipynb

```diff
@@ -0,0 +1,971 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Train Adapt Optimize (TAO) Toolkit](https://developer.nvidia.com/sites/default/files/akamai/embedded-transfer-learning-toolkit-software-stack-1200x670px.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " Transfer learning extracts learned features from an existing neural network into a new one. Transfer learning is often used when creating a large training dataset is not feasible. The goal of this toolkit is to reduce that 80 hour workload to an 8 hour workload, which can enable data scientists to have considerably more train-test iterations in the same time frame.\n",
+    "\n",
+    "Let's see this in action with a use case for the ASR acoustic model."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Automatic Speech Recognition (ASR)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Automatic Speech Recognition (ASR) is often the first step in building a conversational AI model. An ASR model converts audible speech into text. The main metric for these models is to reduce Word Error Rate (WER) while transcribing the text. Simply put, the goal is to take an audio file and transcribe it.\n",
+    "\n",
+    "In this tutorial, we are going to discuss the Conformer-CTC model, which is an end-to-end ASR model that takes in audio and produces text.\n",
+    "\n",
+    "Conformer-CTC supports both character-level and sub-word-level encodings. It employs a combination of self-attention and convolution modules to achieve the best of the two approaches. The self-attention layers support both absolute and relative positional encodings and can learn the global interaction, while the convolutions efficiently capture the local correlations.\n",
+    "\n",
+    "![Conformer-CTC](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/_images/conformer_ctc.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "## ASR using TAO"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Installing and setting up TAO\n",
+    "\n",
+    "Install TAO inside a Python virtual environment. We recommend performing this step first and then launching the tutorial from the virtual environment.\n",
+    "\n",
+    "In addition to installing the TAO Python package, ensure you meet the following software requirements:\n",
+    "\n",
+    "1. `python` 3.6.9\n",
+    "2. `docker-ce` > 19.03.5\n",
+    "3. `docker-API` 1.40\n",
+    "4. `nvidia-container-toolkit` > 1.3.0-1\n",
+    "5. `nvidia-container-runtime` > 3.4.0-1\n",
+    "6. `nvidia-docker2` > 2.5.0-1\n",
```

Comment:
Is there a separate GitHub page for nvidia-docker2?  I only found nvidia-docker and I didn't not find a version 2.5.0-1.  I see 2.5.0 though. I'm looking here: https://github.com/NVIDIA/nvidia-docker/tags?after=v2.7.0-rc.1.

Reply:
Commits [ffebee8](https://github.com/nvidia-riva/tutorials/pull/63/commits/ffebee8acd46e1fde7c03508a7604ac54f79b977) through [386530d](https://github.com/nvidia-riva/tutorials/pull/63/commits/386530dd6a16c6b2d7d37e9ec486b8887a42bd48) should this formatting fix.